documentation/guides/release-and-deploy.md

```diff
@@ -0,0 +1,53 @@
+# Creating Releases
+
+**Note🗒️** The following steps require admin access to the Shopify/quilt github repo.
+
+## 1. Preparing local environment
+
+- Ensure you have the latest `master` branch including all tags:
+
+```
+git checkout master && git pull
+```
+
+**Note** `git pull` should be used instead of `git pull origin master` to ensure that tags are pulled as well.
+
+## 2. Versioning and Tagging
+
+- Begin the release process:
+
+```
+yarn release
+```
+
+- Follow the prompts to choose a version for each package. If you are releasing a new package, we encourage you to version it `1.0.0` to start with.
+
+**Note🗒️** Quilt packages adhere to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+- 🛑 Before the final confirmation. Stop and update the `CHANGELOG.md`.
```

Comment:
Are we able to update the changelog, keep it unstaged, then run `yarn release`. Does `yarn release` pick up and commit those changelog updates. It feels weird to remember to edit the changelog midway through the release script, imo.

Reply:
The reason why I recommend this, is because I find running `yarn release` help me figure out what all the new version number would be before updating `CHANGELOG` (that require accurate version number)

Alternative being someone manually figure out all the version number, update `CHANGELOG`, leave it un stage, and I can change `yarn release` to `git add . && learna publish`

Reply:
Never mind!!
I just tried it and you are able to update `CHANGELOG.md` and stage it before running `lerna publish`. I wont affect the versioning process and it publish just fine!

I will update the doc.

Reply:
> I find running yarn release help me figure out what all the new version number would be 

Fair point, same here tbh. We can always backfill changelogs after the release. 👌